josue/js/app.js

```diff
@@ -0,0 +1,52 @@
+import FormValidator from './Formvalidator.js';
+
+const form = document.querySelector('.form');
+const signInBtn = document.querySelector("#signBtn");
+const registerBtn = document.querySelector("#registerBtn");
+const submitBtn = document.querySelector("#submit");
+const registerFields = document.querySelector("#register");
+let fields = ['name','phone'];
+
+registerFields.remove();
+signInBtn.addEventListener("click",e=>{
+    registerFields.classList.add("register--hidden");
+    registerFields.remove();
+    submitBtn.innerText = "Sign In"
+    fields = ['name','phone']
+    validator.updateFields(fields,"signin")
+
+})
+registerBtn.addEventListener("click",e=>{
+    form.insertBefore(registerFields,submitBtn);
+    registerFields.classList.remove("register--hidden");
+    submitBtn.innerText = "Register"
+    const newFields= ['email','password','password_confirmation']
+    validator.updateFields(newFields,"register")
+});
+submitBtn.addEventListener('click',e=>{
+    console.log("click");
+    e.preventDefault();
+    if(validator.validateOnSubmit()){
```

Comment:
You must store the return value of `validator.validateOnSubmit()` into a variable that tells me what does the boolean return value mean, for example `const isFormValid = validator.validateOnSubmit()` and now you can do `if(isFormValid){...` which is WAY more readable, right?

Reply:
Yes I'll change it :100: